metrics_utility/base/decorators.py

```diff
@@ -26,21 +26,12 @@ def decorate(f):
         f.__insights_analytics_key__ = key
         f.__insights_analytics_version__ = version
         f.__insights_analytics_description__ = description
-        f.__insights_analytics_type__ = format  # CSV/JSON
-        f.__insights_analytics_config__ = config  # config
+        f.__insights_analytics_type__ = format  # 'csv' | 'json' (default)
+        f.__insights_analytics_config__ = config  # True | False (default)
         f.__insights_analytics_fnc_slicing__ = fnc_slicing
         f.__insights_analytics_shipping_group__ = shipping_group
         f.__insights_analytics_full_sync_interval_days__ = full_sync_interval_days
 
         return f
 
     return decorate
-
-
-def slicing(default=False):
```

Comment:
this wasn't used for anything?

Reply:
Nope, `register` with `fnc_slicing=` is used, but `slincing` is not. And no mentions of `__insights_analytics_slicing_name__` or `__insights_analytics_slicing_default__` either.

(I found some more dead code in #112)

Reply:
ok, this might be used by the other collectors (controller, eda), so we might put this back when we move them to this repo